olive/data/component/text_generation.py

```diff
@@ -259,7 +272,7 @@ def text_gen_corpus_pre_process(_dataset, tokenizer, all_kwargs):
                             max_length=args.source_max_len,
                             truncation=True,
                             padding="max_length" if args.pad_to_max_len else False,
-                            add_special_tokens=args.add_special_tokens,
+                            add_special_tokens=False,
```

Comment:
>False [](http://example.com/codeflow?start=47&length=5)

the parent else clause seems aligns with args.corpus_strategy == TextGenCorpusStrategy.LINE_BY_LINE

Reply:
Line 225 applied to both LINE_BY_LINE and LINE_BY_LINE_RANDOM since it's before the if/else. So, the behavior doesn't change.

Reply:
Updated PR title to make this clear